packages/shared/src/components/cards/common/InteractiveFeedCardCover.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import { CardCoverContainer } from './CardCoverContainer';
+import { IconSize } from '../../Icon';
+import { BlockIcon, VIcon } from '../../icons';
+import type { Post } from '../../../graphql/posts';
+import { useInteractiveFeedContext } from '../../../contexts/InteractiveFeedContext';
+
+const InteractiveFeedCardCover = ({ post }: { post: Post }): ReactElement => {
+  const { hidePost, approvePost } = useInteractiveFeedContext();
+  return (
+    <CardCoverContainer title="">
+      <div className="flex gap-2 text-surface-invert">
+        <button
```

Comment:
Why is this a raw button not our Button component?

Reply:
Checked the Figma, looks like they also didn't use our button component. Must be a one-time thing with this large circle buttons.